Widgets_CDAT/textStyleEditorWidget/TextStyleEditor.py

```diff
@@ -0,0 +1,253 @@
+import vcs
+import text_style_preview
+from PySide import QtCore, QtGui
+
+
+class TextStyleEditor(QtGui.QWidget):
+
+    savePressed = QtCore.Signal(str)
+    
+    def __init__(self):
+        super(TextStyleEditor, self).__init__()
+        self.textObject = None
+
+        wrap = QtGui.QVBoxLayout()
+
+        # Set up vertical align
+        self.VAGroup = QtGui.QButtonGroup()
+
+        T_button = QtGui.QPushButton()
```

Comment:
Nitpick: don't name variables with capital letters as the first letter. Syntax highlighting will often think you're talking about a constant.